Lib/sqlite3/__main__.py

```diff
@@ -6,27 +6,31 @@
 from textwrap import dedent
 
 
+def execute(c, sql):
+    try:
+        for row in c.execute(sql):
+            print(row)
+    except sqlite3.Error as e:
+        tp = type(e).__name__
+        try:
+            print(f"{tp} ({e.sqlite_errorname}): {e}")
+        except:
+            print(f"{tp}: {e}")
+    return None
```

Comment:
It is the only `return` in this function. It can be removed.

Reply:
I thought PEP 8 required explicit returns. I might have misremembered; I'll remove those.

Reply:
PEP 8:
> Be consistent in return statements. Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as return None, and an explicit return statement should be present at the end of the function (if reachable):